railties/lib/rails/generators/generated_attribute.rb

```diff
@@ -148,6 +148,7 @@ def default
                      when :references, :belongs_to,
                           :attachment, :attachments,
                           :rich_text                   then nil
+                     when :json                        then {}
```

Comment:
This might require more thought. We really need to return a String, not a Hash.

```suggestion
                     when :json                        then "{\"key\": \"value\"}"
```

But that's pretty cumbersome.